### PR TITLE
Adds in Phobia quirk!

### PR DESCRIPTION
**Pull request by Trilbyspaceclone**
OH GOD
[Changelogs]
Adds in a new bad quirk - Phobia! For three quirk thing points

[why]:
Phobia system is so rare and unused! Lets give it a more main stream task that is not RNG based for once, to even get a phobia.

### CODE REVIEW

File: code/datums/traits/negative.dm [hunk 1]
```diff
@@ -274,3 +274,20 @@
...
+	medical_record_text = "Patient has an extreme or irrational fear of or aversion to something."
+	var/datum/brain_trauma/mild/phobia/phobia
+	
+/datum/quirk/phobia/add()
+	var/mob/living/carbon/human/H = quirk_holder
+	phobia = new
+	H.gain_trauma(phobia, TRAUMA_RESILIENCE_SURGERY)
+
+/datum/quirk/phobia/remove()
+	qdel(phobia)
```
Comment (silicons) [inline code/datums/traits/negative.dm]: QDEL_NULL

File: code/datums/traits/negative.dm [hunk 2]
```diff
@@ -274,3 +274,20 @@
 		dumb_thing = FALSE //only once per life
 		if(prob(1))
 			new/obj/item/reagent_containers/food/snacks/pastatomato(get_turf(H)) //now that's what I call spaghetti code
+
+/datum/quirk/phobia
+	name = "Phobia"
+	desc = "You've had a traumatic past, that has scared you for life while dealing with your greatest fear."
+	value = -3 // It can hardstun you. You can be a job that your phobia targets...
```
Comment (deathride58) [inline code/datums/traits/negative.dm]: This value is pretty high for something that's only shit if you have bad RNG.

**Comment by deathride58** [comment]
Reviews

File: code/datums/traits/negative.dm (hunk 2 above)
Comment (Trilbyspaceclone) [reply code/datums/traits/negative.dm]: Fair lower to 2 seems fire then

**Pull request closed without merging**